src/System.EventLog/src/Resources/Strings.resx

```diff
@@ -0,0 +1,315 @@
+  </data>
+  <data name="MissingLog" xml:space="preserve">
+    <value>Cannot find Log {0} on computer '{1}'.</value>
+  </data>
+  <data name="MissingLogProperty" xml:space="preserve">
+    <value>Log property value has not been specified.</value>
+  </data>
+  <data name="MissingParameter" xml:space="preserve">
+    <value>Must specify value for {0}.</value>
+  </data>
+  <data name="NeedSourceToOpen" xml:space="preserve">
+    <value>Source property was not set before opening the event log in write mode.</value>
+  </data>
+  <data name="NeedSourceToWrite" xml:space="preserve">
+    <value>Source property was not set before writing to the event log.</value>
+  </data>
+  <data name="NoCurrentEntry" xml:space="preserve">
+    <value>No current EventLog entry available, cursor is located before the first or after the last element of the enumeration.</value>
+  </data>
+  <data name="NoLogName" xml:space="preserve">
+    <value>Log to delete was not specified.</value>
+  </data>
+  <data name="ParameterTooLong" xml:space="preserve">
+    <value>The size of {0} is too big. It cannot be longer than {1} characters.</value>
+  </data>
+  <data name="RegKeyMissing" xml:space="preserve">
+    <value>Cannot open registry key {0}\\{1}\\{2} on computer '{3}'.</value>
+  </data>
+  <data name="RegKeyMissingShort" xml:space="preserve">
+    <value>Cannot open registry key {0} on computer {1}.</value>
+  </data>
+  <data name="RegKeyNoAccess" xml:space="preserve">
+    <value>Cannot open registry key {0} on computer {1}. You might not have access.</value>
+  </data>
+  <data name="RentionDaysOutOfRange" xml:space="preserve">
+    <value>'retentionDays' must be between 1 and 365 days.</value>
+  </data>
+  <data name="SomeLogsInaccessible" xml:space="preserve">
+    <value>The source was not found, but some or all event logs could not be searched.  Inaccessible logs: {0}.</value>
+  </data>
+  <data name="SomeLogsInaccessibleToCreate" xml:space="preserve">
+    <value>The source was not found, but some or all event logs could not be searched.  To create the source, you need permission to read all event logs to make sure that the new source name is unique.  Inaccessible logs: {0}.</value>
+  </data>
+  <data name="SourceAlreadyExists" xml:space="preserve">
+    <value>Source {0} already exists on the computer '{1}'.</value>
+  </data>
+  <data name="SourceNotRegistered" xml:space="preserve">
+    <value>The source '{0}' is not registered on machine '{1}', or you do not have write access to the {2} registry key.</value>
+  </data>
+  <data name="TooManyReplacementStrings" xml:space="preserve">
+    <value>The maximum allowed number of replacement strings is 255.</value>
+  </data>
+  <data name="Win2000Required" xml:space="preserve">
```

Comment:
Remove Win2000Required and WinNTRequired strings. Remove the CheckEnvironment and CheckNtenvironment functions where they were used. Remove all calls to those functions.

Basically this was about checking whether it was running on Windows 95/98 or Windows 2000 etc. We are years past caring about that. We only run on Windows 7+. We don't check for that.

Reply:
Removed